src/main/java/org/aksw/simba/squirrel/deduplication/hashing/impl/IntervalBasedMinHashFunction.java

```diff
@@ -0,0 +1,54 @@
+package org.aksw.simba.squirrel.deduplication.hashing.impl;
+
+import org.aksw.simba.squirrel.deduplication.hashing.HashValue;
+import org.aksw.simba.squirrel.deduplication.hashing.RDFHashFunction;
+import org.apache.jena.graph.NodeFactory;
+import org.apache.jena.graph.Triple;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Use Min hash algorithm to compute hash values for triples.
+ */
+public class IntervalBasedMinHashFunction implements RDFHashFunction {
+
+    /**
+     * The number n for the 2^n Intervals for the Hashing.
+     */
+    private int powerNumberOfIntervals;
+
+    /**
+     * Constructor.
+     *
+     * @param powerNumberOfIntervals The powered number of Intervalls for the hashing.
+     */
+    public IntervalBasedMinHashFunction(int powerNumberOfIntervals) {
+        this.powerNumberOfIntervals = powerNumberOfIntervals;
+    }
+
+    @Override
+    public HashValue hash(List<Triple> triples) {
+        List<Integer> listHashValues = new ArrayList<>(powerNumberOfIntervals);
+        for (int i = 0; i < powerNumberOfIntervals; i++) {
+            listHashValues.add(i, null);
```

Comment:
@MatthiasDuerksen why add(i, null) ? what does it mean?

Reply:
@MatthiasDuerksen have you already fixed it?

Reply:
The list of integers is filled with null values so that they can be directly set later on.
In general this is fine but personally, I would prefer the usage of an array since it is faster than a List (even faster than an ArrayList) and the advantages of a list (resizability, etc) are not necessary for the implementation.